src/uucore/src/lib/features/quoting_style.rs

```diff
@@ -316,7 +321,21 @@ pub fn escape_name(name: &OsStr, style: &QuotingStyle) -> String {
             show_control,
         } => {
             let name = name.to_string_lossy();
-            let (quotes, must_quote) = if name.contains(&['"', '`', '$', '\\'][..]) {
+
+            let escaped_chars = [
```

Comment:
could you please move this into a function ? 
wil be easier to reuse in the future

Reply:
Done :+1: